app/controllers/api/v1/events_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::EventsController < ApplicationController
   def index
     Nehra.scrape
-
+    Gmvhra.scrape
```

Comment:
Hey Brendon can you remove the scrapes for the time being? Next we meet I'll be setting up a scheduler to seed the database by running those once a day instead of every API call